tidb-server/main.go

```diff
@@ -318,6 +319,32 @@ func loadConfig() string {
 	return ""
 }
 
+// hotReloadConfigItems lists all config items which support hot-reload.
+var hotReloadConfigItems = []string{"Performance.MaxProcs", "Performance.MaxMemory", "Performance.CrossJoin",
+	"Performance.FeedbackProbability", "Performance.QueryFeedbackLimit", "Performance.PseudoEstimateRatio",
+	"OOMAction", "MemQuotaQuery"}
+
+func reloadConfig(nc, c *config.Config) {
```

Comment:
Should we set OOMAction and MemQuotaQuery in this function?

Reply:
We don't need to do that because they are got from global config directly like `config.GetGlobalConfig().OOMAction` when use.
So after we update the global config pointer by `globalConf.Store(newConf)` in `ReloadGlobalConfig`, their new values are available.

Reply:
We need to add a comment for this.

Reply:
Comments are added.